docs/embedded.md

```diff
@@ -2,41 +2,52 @@
 
 ## Overview
 
-Embedded support is a C#/WinRT feature that allows you to have the WinRT layer of your projection compiled into the projection itself. This is done by setting the build property `CsWinRTEmbedded` to `true` in the app or library's project file. 
+Embedded support is a C#/WinRT feature that allows you to compile the c# projection for the component, within the library/app itself. 
+This is done by setting the build property `CsWinRTEmbedded` to `true` in the app or library's project file. 
 
-Enabling embedded support brings in the WinRT.Runtime sources to the project and automatically sets them to be compiled with the project.
-Any self-contained usage of Windows SDK types must be specified using the property `CsWinRTIncludes`. The specified types will be projected and embedded into the project's *.dll*. This means projects using embedded support no longer have a dependency on `Microsoft.Windows.SDK.NET.dll` and `WinRT.Runtime.dll`.
+Enabling embedded support compiles the sources for WinRT.Runtime with the project.
+For any types from the component which you want to embed in your project, they must be specified using `CsWinRTIncludes`.
+The specified types will be projected and embedded into the project's *.dll*. 
+This means projects using embedded support no longer have a dependency on a projection for the component, `WinRT.Runtime.dll` or `Microsoft.Windows.SDK.NET.dll` in the case of the Windows SDK projection.
 
 For an example, you can look at the [sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/TestEmbedded). 
 
+Embedded support introduces new features to the C#/WinRT toolchain:

```

Comment:
add some cavet here like if your use of WinRT types is scoped to your binary or something like that.